RealmSwift-swift2.2/List.swift

```diff
@@ -426,30 +422,32 @@ public final class List<T: Object>: ListBase {
      As a result, the initial notification will reflect the state of the Realm
      after the write transaction, and will not include change information.
 
-         let person = realm.objects(Person).first!
-         print("dogs.count: \(person.dogs.count)") // => 0
-         let token = person.dogs.addNotificationBlock { (changes: RealmCollectionChange) in
-             switch changes {
-                 case .Initial(let dogs):
-                     // Will print "dogs.count: 1"
-                     print("dogs.count: \(dogs.count)")
-                     break
-                 case .Update:
-                     // Will not be hit in this example
-                     break
-                 case .Error:
-                     break
-             }
-         }
-         try! realm.write {
-             let dog = Dog()
-             dog.name = "Rex"
-             person.dogs.append(dog)
+     ```swift
+     let person = realm.objects(Person).first!
+     print("dogs.count: \(person.dogs.count)") // => 0
+     let token = person.dogs.addNotificationBlock { (changes: RealmCollectionChange) in
```

Comment:
it's no longer necessary to explicitly state the parameter type here: `{ changes in`